### PR TITLE
V01-28: put console.log function to the file

### PR DESCRIPTION
**Pull request by meVeri**

### CODE REVIEW

File: OldWeb/oldweb.html [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
-		<meta charset="UTF-8" />
-		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
-		<title>My Old-fashion WebSite</title>
-	</head>
-	<body>
-		yes
-	</body>
-</html>
+console.log({a: 1})
+console.log(a)
```
Comment (theXaMi) [inline OldWeb/oldweb.html]: This needs to be done in JavaScript, not HTML. Here it is just a text. You can put JavaScript here using `<script>` tag, or you can do it in the `index.tsx` file.

**Review by theXaMi — CHANGES_REQUESTED** [review]
I see that you changed the formatting of the entire file. Why is that? 😛

File: OldWeb/oldweb.html (hunk 1 above)
Comment (meVeri) [reply OldWeb/oldweb.html]: both lines, or just one?

**Comment by meVeri** [comment]
> I see that you changed the formatting of the entire file. Why is that? 😛

Just for test, no reason for that, but i can change it if u want xd

**Comment by meVeri** [comment]
Ready to Rewiev

**Comment by theXaMi** [comment]
`console.log()`s are looking good, but commits are terrible. The `D3D` and `asdasdasd` files are not a part of the task, please remove them and squash the commits.

**Review by theXaMi — CHANGES_REQUESTED** [review]
Please fix this commit mess

File: OldWeb/oldweb.html (hunk 1 above)
Comment (theXaMi) [reply OldWeb/oldweb.html]: Both of them 😛

**Review by theXaMi — APPROVED** [review]
:+1:

**Review by theXaMi — APPROVED** [review]

**Pull request merged**